### ISSUE TITLE
results

### ISSUE THREAD

**Issue by wesbos** [wesbos/ff-me#1]

**Comment by wesbos** [comment]
```bash
npm start

> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.17.0
  Apple M1 Max 2.4ghz 10 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
[1]    10543 bus error  npm start
```

**Comment by r3volution11** [comment]
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v19.2.0
  Apple M1 Pro 2.4ghz 10 cores
  macOS 13.3.1 (a) macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
zsh: bus error  npm start

**Comment by nicokaag** [comment]
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.16.1
  Intel Core™ i7-4770HQ 2.2ghz 8 cores
  macOS 12.6.7 macOS Monterey

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
zsh: bus error  npm start

**Comment by wridgeu** [comment]
```shell
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.12.0
  Intel Gen Intel® Core™ i7-12700K 3.6ghz 20 cores
  Microsoft Windows 11 Pro 10.0.22621

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
```
Here with a more recent LTS node version v18.17.0.

```shell
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.17.0
  Intel Gen Intel® Core™ i7-12700K 3.6ghz 20 cores
  Microsoft Windows 11 Pro 10.0.22621

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
```

Same output with latest v20.5.0.

```shell
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v20.5.0
  Intel Gen Intel® Core™ i7-12700K 3.6ghz 20 cores
  Microsoft Windows 11 Pro 10.0.22621

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
```
----

Hope it helps, somehow. :)

**Comment by arthurdotwork** [comment]
```
ffmpeg-delete-me@0.0.1 start
node test.js


  Node v18.13.0
  Apple M1 Pro 2.4ghz 8 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
[1]    51564 bus error  npm run start
```

**Comment by claudiorivera** [comment]
```bash
npm start

> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.16.1
  Intel Core™ i7-9750H 2.6ghz 12 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
[1]    80233 bus error  npm start
```

**Comment by jcs224** [comment]
Linux with Intel

```
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.14.0
  Intel Gen Intel® Core™ i7-1165G7 2.8ghz 8 cores
  Pop 22.04 LTS jammy

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
```

**Comment by wesbos** [comment]
Seems its only Macs, but both intel and apple silicon

**Comment by Three97** [comment]
```
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.14.2
  Apple M1 Max 2.4ghz 10 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
[1]    16554 bus error  npm start
```

**Comment by Moshyfawn** [comment]
```bash
npm start

> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.17.0
  Apple M1 2.4ghz 8 cores
  macOS 14.0 macOS Sonoma

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
[1]    82034 bus error  npm start
```

**Comment by kevin-lynch** [comment]
> ffmpeg-delete-me@0.0.1 start /Users/kevinlynch/git_hub_test/ff-me
> node test.js


Node v14.15.4
Intel Core™ i7-8850H 2.6ghz 12 cores
macOS 13.4.1 (c) macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
(node:20693) UnhandledPromiseRejectionWarning: Error: bad memory
at /Users/kevinlynch/git_hub_test/ff-me/node_modules/@ffmpeg.wasm/core-mt/dist/core.js:15:1
at Function.<anonymous> (/Users/kevinlynch/git_hub_test/ff-me/node_modules/@ffmpeg.wasm/main/dist/index.js:165:64)
at Generator.next (<anonymous>)
at fulfilled (/Users/kevinlynch/git_hub_test/ff-me/node_modules/@ffmpeg.wasm/main/dist/index.js:28:22)
at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:20693) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:20693) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by Joehoel** [comment]
```sh
npm start

> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v16.17.1
  Apple M1 Pro 2.4ghz 10 cores
  macOS 13.4.1 (c) macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
/var/folders/8l/1p1bzg196_3cqynqtxckg4v40000gn/T/start-bce84cc2.sh: line 1: 21155 Bus error: 10           node test.js```

**Comment by DreamOfIce** [comment]
It looks like this issue has been fixed in the latest v20.5.0:
```sh
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v20.5.0
  Apple M1 Pro (Virtual) 2.4ghz 4 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
```
But not yet fixed in v18.17.0 LTS:

``` sh
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.17.0
  Apple M1 Pro (Virtual) 2.4ghz 4 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
zsh: bus error  npm start
```

**Comment by Siddhant-K-code** [comment]
In Node 19:

```bash

> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v19.3.0
  Apple M1 Pro 2.4ghz 8 cores
  macOS 13.5 macOS Ventura

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
zsh: bus error  npm start

```

**Comment by pexeixv** [comment]
```
> ffmpeg-delete-me@0.0.1 start D:\alt\ff-me
> node test.js


  Node v18.15.0
  Intel Gen Intel® Core™ i3-1115G4 3ghz 4 cores
  Microsoft Windows 10 Home Single Language 10.0.19045

debug Import '@ffmpeg.wasm/core-mt' with cjs require()
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by Seefer** [comment]
> ffmpeg-delete-me@0.0.1 start
> node test.js


  Node v18.16.1
  Intel Core™ i9-10900K 3.7ghz 20 cores
  Microsoft Windows 11 Pro 10.0.22621 

debug Import '@ffmpeg.wasm/core-mt' with cjs require()

**Comment by wesbos** [comment]
Thanks everyone - super helpful! Seems we got to the bottom of it and it's fixed in the latest node 20.5.0.
